hooks/RoboFile.php

```diff
@@ -55,6 +55,17 @@ public function setupSkeleton()
         $config->extra->{"enable-patching"} = 'true';
         $config->extra->{"patches"} = new \stdClass();
         file_put_contents('composer.json', json_encode($config));
+
+        // Create a directory for our artifacts.
+        $this->taskFilesystemStack()
+          ->mkdir('artifacts')
+          ->mkdir('artifacts/phpunit')
+          ->mkdir('artifacts/phpcs')
+          ->run();
+
+        $this->taskFilesystemStack()
+          ->chown('artifacts', 'www-data', TRUE)
```

Comment:
This is so you don't have to do this at the circle config file?

Reply:
Right, and so it's created for every job. I figure even if you aren't using circle having a standard artifacts directory is helpful.

Reply:
Gotcha. I like that.